doc/website/getting-started/overview.md

```diff
@@ -0,0 +1,540 @@
+# Getting started with iceoryx
+This document covers the core functionality of the ``iceoryx`` middleware and is intended to quickly get started to
+set up iceoryx applications. It is no in-depth API documentation and while the API is still subject to changes, the
+basic concepts presented here will still apply.
+## General
+To set up a collection of applications using iceoryx (an ``iceoryx system``), the applications need to initialize a
+runtime and create ``publishers`` and ``subscribers``. The publishers send data of a specific ``topic`` which can be
+received by subscribers of the same topic. To enable publishers to offer their topic and subscribers to subscribe to
+these offered topics, the middleware daemon, called ``Roudi``, must be running.
+
+For further information how iceoryx can be used see the [examples](../iceoryx_examples/README.md). The
+[conceptual-guide.md](../../conceptual-guide.md) provides additional information about the ``Shared Memory
+communication`` that lies at the heart of iceoryx.
+
+We now briefly define the main entities of an iceoryx system before showing how they are created and used by the
+iceoryx API.
+### Roudi
+
+Roudi is an abbreviation for **Rou**ting and **Di**scovery. Roudi takes care of the
+communication setup but does not actually participate in the communication between the publisher and the subscriber.
+RouDi can be thought of as the switchboard operator of iceoryx. One of his other major tasks is the setup of the
+shared memory, which the applications use for exchanging payload data. Sometimes referred to as daemon, RouDi manages
+the shared memory and is responsible for the service discovery, i.e. enabling subscribers to find topics offered by publishers. It also keeps track of all applications which have initialized a runtime and are hence able to use
+publishers or subscribers. To view the available command line options call `iox-roudi --help`.
+
+### Runtime
+
+Each application which wants to use iceoryx has to instantiate its runtime, which essentially enables communication
+with Roudi. Only one runtime object per user process is allowed.
+
+To do so, the following lines of code are required
+
+    #include "iceoryx_posh/runtime/posh_runtime.hpp"
+
+    iox::runtime::PoshRuntime::initRuntime("some_unique_application_name");
+
+### Topics
+
+A topic in iceoryx defines the data to be transmitted and is uniquely identified by three string identifiers.
+
+1. ``Group`` name
+2. ``Instance`` name
+3. ``Topic`` name
+
+A triple consisting of such strings is called a ``service description``. In Autosar terminology these three
```

Comment:
Could you use the class name here `ServiceDescription`?

Reply:
Sure, but I will omit the namespaces.